mst/aiven-client.md/aiven-client-install.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Aiven Client for Managed Service for TimescaleDB
+excerpt: Install and configure Aiven Command Line tool for fully managed services on AWS, Azure, or GCP.
+product: mst
+---
+
+# Aiven Client for fully managed services
+
+Aiven Client is a command line tool for fully managed services. To use Aiven
+Client you need to first create an authentication token and then configure it to
+connect to your Managed Service for TimescaleDB using the command line.
+
+## Create an authentication token in Managed Service for TimescaleDB
+
+To connect to Managed Service for TimecaleDB using Aiven Client you need to
+create an authentication token.
+
+<procedure>
+
+### Creating an authentication token in Managed Service for TimescaleDB
+
+1.  Log in to your [Managed Service for TimescaleDB portal][mst-login].
+1.  Click `User Information` in the top right corner.
+1.  In the `User Profile` page, navigate to the `Authentication`tab.
+1.  Click `Generate Token`.
+1.  In the `Generate access token` dialog, type a descriptive name for the
+    token and leave the rest of the fields blank.
+1.  Copy the generated authentication token and save it.
+
+</procedure>
+
+## Install the Aiven Client
+
+[Aiven Client][aiven-github] is provided as a Python package, so you can install
+it on a Linux, MacOS, or Windows system using `pip`, if you have already installed Python. 
+
+Use this command: 
+
+```bash
+pip install aiven-client
+```
+
+## Configure Aiven Client to connect to Managed Service for TimescaleDB
+
+To access Managed Service for TimescaleDB with the Aiven Client, you need an 
+authentication token. Aiven Client uses this authentication token to access your 
+services on Managed Service for TimescaleDB.
+
+<procedure>
+
+### Configuring Aiven Client to connect to Managed Service for TimescaleDB
+
+1.  Change to the install directory that contains the configuration files:
+
+    ```bash
+    cd ~/.config/aiven/
+    ```
+
+1.  Open the `aiven-credentials.json` using any editor and update these
+    lines with your Managed Service for TimescaleDB `User email`, and the
+    `authentication token` that you generated :
+
+    ```bash
+    {
+    "auth_token": "ABC1+123...TOKEN==",
+    "user_email": "your.email@timescale.com"
+    }
+    ```
+
+1.  Save the `aiven-credentials.json` file.
+
+1.  To verify that you can access your services on Managed Service for
+    TimescaleDB, type:
+
+    ```bash
+    avn project list
+    ```
+
+    This command shows a list of all your projects:
+
+    ```bash
+    
+    PROJECT_NAME       DEFAULT_CLOUD            CREDIT_CARD
+    =================  =======================  ===========
```

Comment:
Might need to adjust these to line up as well.

Reply:
adjusted